plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -0,0 +1,270 @@
+
+
+### Prepare remote databases
+Follow the instructions to [prepare remote mongo](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-mongo) and [prepare remote postgres](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-postgres) to make the remote database servers ready for installation.
+
+### Run the installer
+To install and configure Katello we just need to run 
+```
+  foreman-installer --scenario katello \
+    --foreman-db-host postgres.example.com \ 
+    --foreman-db-password <FOREMAN_PASSWORD> \
+    --foreman-db-database foreman \
+    --katello-candlepin-db-host postgres.example.com \
+    --katello-candlepin-db-name candlepin \
+    --katello-candlepin-db-password <CANDLEPN_PASSWORD> \
+    --katello-candlepin-manage-db false \
+    --katello-pulp-db-username pulp \ 
+    --katello-pulp-db-password <PULP_PASSWORD> \ 
+    --katello-pulp-db-seeds “mongo.example.com:27017” \
+    --katello-pulp-db-name pulp_database
+```
+
+Note: for more related options and tips on SSL configuration see [Full list of options](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#full-list-of-remote-database-related-options-in-the-installer)
+
+## Migration of existing Katello
+We assume that Katello was installed and is running on `katello.example.com`.
+
+### Prepare remote databases
+Follow the instructions to [prepare remote mongo](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-mongo) and [prepare remote postgres](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-postgres) to make the remote database servers ready for migration.
+
+### Stop the Katello server
+Stop the Katello related services to minimize risk of the data changes during the migration
+```
+  katello-service stop
+  systemctl start postgresql
+  systemctl start mongod
+```
+
+### Dump databases
+Dump the local databases
+```
+  foreman-maintain backup online --skip-pulp-content --preserve-directory -y /tmp/migration_backup
+```
+
+### Restore data in remote databases
+You can restore the SQL dumps to the remote databases from the katello system.
+
+```
+PGPASSWORD='<FOREMAN_PASSWORD>' pg_restore -h postgres.example.com -U foreman -d foreman < /tmp/migration_backup/foreman.dump
+PGPASSWORD='<CANDLEPIN_PASSWORD>' pg_restore -h postgres.example.com -U candlepin -d candlepin < /tmp/migration_backup/candlepin.dump
+mongorestore --host mongo.example.com --db pulp_database --username pulp --password <PULP_PASSWORD> /tmp/migration_backup/mongo_dump
+```
+Now the copy of the local database is also at the remote locations.
+
+### Update the configuration
```

Comment:
I don't really see why you rerun the installer now with additional options rather than doing it right the first time.

Reply:
This is for an existing installation of Katello, this doc is split into two parts, new installs and existing installs